nalgebra-glm/src/geometric.rs

```diff
@@ -4,7 +4,7 @@ use crate::aliases::{TVec, TVec3};
 use crate::traits::{Alloc, Dimension, Number};
 
 /// The cross product of two vectors.
-pub fn cross<N: Number, D: Dimension>(x: &TVec3<N>, y: &TVec3<N>) -> TVec3<N> {
+pub fn cross<N: Number>(x: &TVec3<N>, y: &TVec3<N>) -> TVec3<N> {
```

Comment:
Isn't this a breaking change?

Reply:
It is, yes. Though it's a breaking change for nalgebra-glm, not nalgebra itself.